src/test/java/htsjdk/samtools/CRAMComplianceTest.java

```diff
@@ -35,6 +34,7 @@
     @DataProvider(name = "partialVerification")
     public Object[][] getPartialVerificationData() {
         return new Object[][] {
+                {"amb#amb"},
```

Comment:
amb#amb.sam contains non-IUPAC read bases that don't survive BAM or CRAM roundtrip. Since the comparison in the test is done against the original SAM records the fullVerificationTest would fail.